RealmSwift/MongoClient.swift

```diff
@@ -178,28 +193,28 @@ public typealias MongoDeleteBlock = RLMMongoDeleteBlock
 public typealias MongoCollection = RLMMongoCollection
 
 extension MongoCollection {
-    
+
     /// Encodes the provided value to BSON and inserts it. If the value is missing an identifier, one will be
     /// generated for it.
     /// - Parameters:
     ///   - document: document  A `Document` value to insert.
     ///   - completion: The result of attempting to perform the insert. An Id will be returned for the inserted object on sucess
     public func insertOne(_ document: Document, _ completion: @escaping MongoInsertBlock) {
-        guard let rlmBSON = ObjectiveCSupport.convert(object: AnyBSON(document)) as? [String : RLMBSON] else {
+        guard let rlmBSON = ObjectiveCSupport.convert(object: AnyBSON(document)) as? [String: RLMBSON] else {
             return completion(nil, BSONError.swiftToObjCConversion)
         }
         self.__insertOneDocument(rlmBSON, completion: completion)
     }
-    
+
     /// Encodes the provided values to BSON and inserts them. If any values are missing identifiers,
     /// they will be generated.
     /// - Parameters:
     ///   - documents: The `Document` values in a bson array to insert.
     ///   - completion: The result of the insert, returns an array inserted document ids in order.
     public func insertMany(_ documents: [Document], _ completion: @escaping MongoInsertManyBlock) {
         do {
-            let convertedDocuments = try documents.map { (document: Document) -> [String : RLMBSON] in
-                guard let rlmBSON = ObjectiveCSupport.convert(object: AnyBSON(document)) as? [String : RLMBSON] else {
+            let convertedDocuments = try documents.map { (document: Document) -> [String: RLMBSON] in
```

Comment:
Is this actually needed? Shouldn't you just be able to convert the whole array?

Reply:
Changed to: `        let bson = ObjectiveCSupport.convert(object: .array(documents.map {.document($0)}))
`